applications/audio/broadcast_configuration_tool/src/main.c

```diff
@@ -0,0 +1,504 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/shell/shell.h>
+#include <nrfx_clock.h>
+
+#include "broadcast_source.h"
+#include "zbus_common.h"
+#include "macros_common.h"
+#include "bt_mgmt.h"
+#include "sd_card.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(main, CONFIG_MAIN_LOG_LEVEL);
+
+ZBUS_CHAN_DECLARE(bt_mgmt_chan);
+ZBUS_CHAN_DECLARE(sdu_ref_chan);
+
+struct bt_le_ext_adv *ext_adv;
+
+/* Buffer for the UUIDs. */
+#define EXT_ADV_UUID_BUF_SIZE (128)
+NET_BUF_SIMPLE_DEFINE_STATIC(uuid_data, EXT_ADV_UUID_BUF_SIZE);
+NET_BUF_SIMPLE_DEFINE_STATIC(uuid_data2, EXT_ADV_UUID_BUF_SIZE);
```

Comment:
This is just allocating the memory for it though, not actually advertising both sets. So I guess it will be part of a README as some point what the limits are